hvantk/htables/expression_atlas.py

```
@@ -0,0 +1,286 @@
+    # Write the Hail Table to a file
+    if output_file is not None:
+        ht = ht.checkpoint(output_file, overwrite=overwrite)
+
+    return ht
+
+
+def create_mt_from_expression_atlas_matrix(
+    expression_matrix_path: str,
+    output_path: str = None,
+    delimiter: str = "\t",
+    row_fields: dict = {"Gene ID": hl.tstr, "Gene Name": hl.tstr, "GeneID": hl.tstr},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)